tuner/tuner/dispatch_parser.py

```diff
@@ -77,6 +78,19 @@ def parse_mlir(mlir_text: str, ctx: TunerContext) -> ir.Module:
     return mlir_module
 
 
+def find_root_op(
```

Comment:
We should be able to use the root op attribute that IREE emits now: https://github.com/iree-org/iree/pull/19345

Reply:
Cool, I made use of the root op in the matchers now, which simplified things quite a bit.